src/reducers/tasksReducer.js

```diff
@@ -52,6 +52,18 @@ export default function(state = initialState, action) {
         })
       });
 
+    case 'PATCH_TASK_PENDING':
+      return state;
+
+    case 'PATCH_TASK_FULFILLED':
+      const patchedTask = action.payload;
+      patchedTask.subtask_gids = state.taskMap[action.payload.gid];
```

Comment:
@Moe-Shoman I believe

`patchedTask.subtask_gids = state.taskMap[action.payload.gid];`

should be

`patchedTask.subtask_gids = state.taskMap[patchedTask.gid].subtask_gids || [];`

Reply:
I made this tweak in ee7f5673b877883903fda71f52274230f9e6026e